pkg/core/resources/resources.go

```diff
@@ -1,10 +1,6 @@
-package helper
+package resources
```

Comment:
As I see these `Resources` are used only in Backend, so does it make sense to leave it in the `core`? This thing has very specific purpose.